frontend_lib/src/component/Avatar/Avatar.jsx

```diff
@@ -45,44 +46,58 @@ export class Avatar extends React.Component {
   render () {
     const { props } = this
 
-    const generatedColor = this.generateColorFromName(props.publicName)
+    const publicName = props.user.publicName || props.user.public_name
+    const hasAvatar = Object.keys(props.user).includes('hasAvatar') ? props.user.hasAvatar : props.user.has_avatar
+    const letterAvatar = publicName ? this.getTwoLetters(publicName.toUpperCase()) : '?'
+    const sizeAsNumber = parseInt(props.size.replace('px', ''))
+
+    const generatedColor = publicName ? this.generateColorFromName(publicName) : '#000'
```

Comment:
We have the same check at `generateColorFromName`, I don't mind that it is done here, but please use the same default color.